templates/_layout.html

```diff
@@ -14,7 +14,7 @@
 {# Output
 {# --------------------------------------------------------- #}
 <!DOCTYPE html>
-<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" class="no-js">
+<html xmlns="http://www.w3.org/1999/xhtml" lang="{{ craft.app.language }}" class="no-js">
```

Comment:
I think we want to use `currentSite.language` here, otherwise I think we're referring to the installed language of Craft

```suggestion
<html xmlns="http://www.w3.org/1999/xhtml" lang="{{ currentSite.language }}" class="no-js">
```

Reply:
Actually looks like both will work. Not sure if they're actually different, or just aliases to each other.